packages/ios-client/Sources/FishjamClient/FishjamClientInternal.swift

```diff
@@ -345,14 +345,15 @@ class FishjamClientInternal {
         self.webSocket?.write(data: peerMessage)
     }
 
-    private func receiveEvent(event: SerializedMediaEvent) {
-        rtcEngineCommunication.onEvent(serializedEvent: event)
+    private func receiveEvent(event: Fishjam_MediaEvents_Server_MediaEvent) {
+        rtcEngineCommunication.onEvent(event: event)
     }
 
     func websocketDidConnect() {
         let authRequest = Fishjam_PeerMessage.with({
             $0.authRequest = Fishjam_PeerMessage.AuthRequest.with({
                 $0.token = self.config?.token ?? ""
+                $0.sdkVersion = "mobile-0.6.0"
```

Comment:
Could it be read from config file?

Reply:
🫡 https://linear.app/swmansion/issue/FCE-952